gcc/testsuite/rust/compile/macro-issue2264.rs

```diff
@@ -0,0 +1,6 @@
+macro_rules! m {
```

Comment:
Can you add a similar test with identifiers instead of integer literals?

Reply:
Modified the test